src/openklant/components/klantinteracties/api/serializers/internetaken.py

```diff
@@ -89,21 +149,33 @@ def validate(self, attrs):
 
     @transaction.atomic
     def create(self, validated_data):
-        actor_uuid = str(validated_data.pop("actor").get("uuid"))
+        actoren = validated_data.pop("actoren", None)
         klantcontact_uuid = str(validated_data.pop("klantcontact").get("uuid"))
 
-        validated_data["actor"] = Actor.objects.get(uuid=actor_uuid)
         validated_data["klantcontact"] = Klantcontact.objects.get(
             uuid=klantcontact_uuid
         )
 
-        return super().create(validated_data)
+        internetaak = super().create(validated_data)
+        if actoren:
+            bulk_create_instances = [
+                InterneActorenThoughModel(internetaak=internetaak, actor=actor)
+                for actor in self._get_actoren(actoren)
+            ]
+            InterneActorenThoughModel.objects.bulk_create(bulk_create_instances)
+
+        return internetaak
 
     @transaction.atomic
     def update(self, instance, validated_data):
-        if "actor" in validated_data:
-            if actor := validated_data.pop("actor", None):
-                validated_data["actor"] = Actor.objects.get(uuid=str(actor.get("uuid")))
+        if "actoren" in validated_data:
+            actoren = validated_data.pop("actoren")
+            instance.actoren.clear()
```

Comment:
Always delete and create relations seems unnecessary. I think it's better to compare first that old and new actors are not the same

Reply:
This will be done in the future but because we currently are bound to a temporary ordering model. I think it is not worth the time and resources to create the functionality of comparing the data between the existing data and the user submitted data and compare the order of both.

Deleting the relations and creating them afterwards will guarantee that the order will always be correct. So I think that for now this is a fine solution.

Reply:
Ok, let's keep it until we get rid of the deprecated field and OrderedModel

Reply:
Is it possible to modify it now?